### DISCUSSION TITLE
RHS

### DISCUSSION THREAD

**Discussion by Zerathrax** [Vdauphin/HeartsAndMinds#1099] [Question and answer]
How do you get RHS to work with this? I tried following the wiki but it's very confusing to me, and I can't seem to find anything else on this subject. I see RHS stuff in the files but am failing to see how to make it the faction for the player. I only see enemy faction in parameters in game, and just having the mods enabled is not changing the player faction.

**Comment by Vdauphin** [comment]
Hello,

> how to make it the faction for the player.

Well, just change player's vehicles... Follow this http://vdauphin.github.io/HeartsAndMinds/Add-vehicle-or-boat-to-H&M-system#1-by-mission-editing
And, for player, use the Arsenal to have the correct loadout...
 
> I only see enemy faction in parameters in game, and just having the mods enabled is not changing the player faction.

As you are saying this parameter is to change enemy faction, so not player faction.

Cheers

**Reply by Zerathrax** [reply]
Sorry for such a late reply but, I did what you said the other day. I swapped out the vehicles for RHS ones, and it works, but now the server won't save...

Params are set right, and even when you manually save via self ace menu, it just says "Saving in progress... please wait" then nothing. It never says complete, failed, or anything. I even reset the whole server so it has to be something with the PBO but all I did was swap vehicles how did this effect saving?

**Reply by GoldJohnKing** [reply]
> Sorry for such a late reply but, I did what you said the other day. I swapped out the vehicles for RHS ones, and it works, but now the server won't save...
> 
> Params are set right, and even when you manually save via self ace menu, it just says "Saving in progress... please wait" then nothing. It never says complete, failed, or anything. I even reset the whole server so it has to be something with the PBO but all I did was swap vehicles how did this effect saving?

You might missing some procedures in the docs, or misconfigure those, see the italics below:

http://vdauphin.github.io/HeartsAndMinds/Add-vehicle-or-boat-to-H&M-system#1-by-mission-editing

1: By mission editing:
You can add new vehicle in the mision.sqm :

Open the mission in the editor (mission.sqm).
Add vehicles.
_Give a unique name like: btc_veh_16,btc_veh_17 …_
Add in the init field : this setVariable ["btc_dont_delete",true]
Save.
_Open the /core/def/mission.sqf L285
Add names you gave to your vehicles. (Also, you have to delete corresponding vehicles' unique name that you've deleted in mission.sqm)_
Save.

Or else the mission won't run right.

Or you should upload your mission file here.

**Reply by Zerathrax** [reply]
I was just about to write that I am an idiot and found the issue. I accidentally put BTS instead of BTC for btc_veh_# so it was bts_veh_# on 2 of the vehicles. I just fixed it after noticing and repacked, and its saving now.

For anyone who finds this later - double check your spelling lol.
